lib/index.js

```diff
@@ -0,0 +1,143 @@
+async function goHome() {
+  await browser.goTo('#Shell-home');
+  await browser.waitForUI5();
+}
+
+async function pressTile(name) {
+  const control = await browser.asControl({
+    selector: {
+      controlType: "sap.m.GenericTile",
+      properties: {
+        header: name
+      }
+    }
+  });
+  await control.press()
+}
+
+async function searchFor(text) {
+  await browser.asControl({
+    selector: {
+      controlType: "sap.m.SearchField",
+      interaction: {
+        idSuffix: "I"
+      }
+    }
+  }).enterText(text)
+}
+
+async function performStandardAction( name ) {  
+  const control = await browser.asControl({
+    selector: {
+      controlType: "sap.m.Button",
+      interactable: true,
+      id: new RegExp(`.*\:\:StandardAction\:\:${name}\$`)
+    }
+  });
+  await control.press();
+}
+
+async function setFieldValue( name, value ) {
+  const control = await browser.asControl({
+    selector: {
+      controlType: "sap.m.Input",
+      interaction: "root",
+      labelFor: {
+        text: name
+      }
+    }
+  });
+  await control.setValue(value);
+}
+
+async function openValueHelpForField( name ) {
+  const control = await browser.asControl({
+    selector: {
+      controlType: "sap.m.Input",
+      labelFor: {
+        text: name
+      }
+    }
+  });
+  await control.press();
+}
+
+async function _findTableWithTitle(title, controlType, opt={}) {
+  let { searchOpenDialogs } = opt;
```

Comment:
not sure about your intended usage of `let` and `const` here and this file in general. Personally, I prefer `const` over `let` for immutable var assignments, but that's just me. Please just be aware of it.

Reply:
changed that: let it be const ;)